ru/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -322,127 +322,127 @@
 <!-- topic context menu -->
 <!ENTITY pbTopicContextMenu-edit.label         "Изменить">
 <!ENTITY pbTopicContextMenu-edit.accesskey     "E">
-<!ENTITY pbTopicContextMenu-favorite.label     "Любимая">
+<!ENTITY pbTopicContextMenu-favorite.label     "Избранная метка">
 <!ENTITY pbTopicContextMenu-favorite.accesskey "F">
 <!ENTITY pbTopicContextMenu-remove.label       "Удалить">
 <!ENTITY pbTopicContextMenu-remove.accesskey   "R">
-<!ENTITY pbTopicContextMenu-editAll.label      "Редактировать метки...">
+<!ENTITY pbTopicContextMenu-editAll.label      "Редактировать метки…">
 <!ENTITY pbTopicContextMenu-editAll.accesskey  "t">
 
 <!-- bubble context menu XXX TODO ACCESS KEYS-->
-<!ENTITY pbBubbleContextMenu-remove.label      "Remove Address">
-<!ENTITY pbBubbleContextMenu-copy.label        "Copy Address">
-<!ENTITY pbBubbleContextMenu-edit.label        "Edit Address">
-<!ENTITY pbBubbleContextMenu-open.label        "Open in Address Book">
-<!ENTITY pbBubbleContextMenu-add.label         "Add to Address Book">
+<!ENTITY pbBubbleContextMenu-remove.label      "Удалить адрес">
+<!ENTITY pbBubbleContextMenu-copy.label        "Копировать адрес">
+<!ENTITY pbBubbleContextMenu-edit.label        "Изменить адрес">
+<!ENTITY pbBubbleContextMenu-open.label        "Открыть в адресной книге">
+<!ENTITY pbBubbleContextMenu-add.label         "Добавить в адресную книгу">
 
-<!ENTITY pbInsertHTMLToolbarCmd.label "HTML">
-<!ENTITY pbRemoveTextToolbarCmd.label "Remove text styles">
+<!ENTITY pbInsertHTMLToolbarCmd.label "Вставить или редактировать HTML">
```

Comment:
CORRECTIONS NEEDED for line 339:
+<!ENTITY pbInsertHTMLToolbarCmd.label "HTML">